plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/GameStateMachine.hs

```diff
@@ -6,7 +6,9 @@
 {-# LANGUAGE DataKinds         #-}
 {-# LANGUAGE ViewPatterns      #-}
 {-# LANGUAGE NoImplicitPrelude #-}
+{-# OPTIONS_GHC -fno-strictness #-}
```

Comment:
Why?

Reply:
We need to scatter these around to prevent GHC unboxing things, it's annoying.